docs/_static/spec/server.yml

```diff
@@ -0,0 +1,1157 @@
+openapi: "3.0.1"
+info:
+  title: "Rasa Core - Server Endpoints"
+  version: "0.12.0"
+  description: >-
+    The Rasa Core server provides endpoints to retrieve trackers of
+    conversations as well as endpoints to modify them.
+servers:
+  - url: "http://localhost:5005"
+    description: "Local development server"
+x-reuse:
+  tracker:
+    &tracker-example
+    sender_id: default
+    latest_event_time: 1537645578.314389
+    latest_input_channel: rest
+    paused: false
+    slots: {}
+    followup_action:
+    latest_message:
+      text: hello
+      entities: []
+      intent:
+        confidence: 0.57
+        name: greet
+      intent_ranking:
+      - confidence: 0.57
+        name: greet
+      - confidence: 0.33
+        name: goodbye
+    events:
+    - event: action
+      name: action_listen
+      timestamp: 1537645578.274815
+    - event: user
+      input_channel: rest
+      text: hello
+      timestamp: 1537645578.314389
+      parse_data:
+        text: hello
+        entities: []
+        intent:
+          confidence: 0.57
+          name: greet
+        intent_ranking:
+        - confidence: 0.57
+          name: greet
+        - confidence: 0.33
+          name: goodbye
+
+paths:
+  /:
+    get:
+      tags:
+      - Status
+      summary: Say Hello to Core
+      description: >-
+        This URL can be used as an endpoint to run
+        health checks against. When Core is running
+        this will return a 200.
+      operationId: hello
+      x-code-samples:
+        - lang: shell
+          source: curl http://localhost:8000
+      responses:
+        200:
+          description: Up and running
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: hello text
+              example: >-
+                hello from Rasa Core: 0.12.0
+
+  /version:
+    get:
+      tags:
+      - Status
+      summary: Meta Info
+      description: >-
+        Metadata about the running Core instance.
+      operationId: version
+      responses:
+        200:
+          description: Version of Core
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  version:
+                    type: string
+                    description: >-
+                      Core version number
+                  minimum_compatible_version:
+                    type: string
+                    description: >-
+                      Minimum version this Core is
+                      able to load models from
+              example:
+                version": 0.12.0
+                minimum_compatible_version: 0.9.0
```

Comment:
I know it's just an example but let's still put 0.12 here because it might be confusing